router/image/conf.d/nginx.conf.toml

```diff
@@ -12,6 +12,7 @@ keys = [
   "/deis/builder",
   "/deis/store/gateway",
   "/deis/certs",
+  "/registry/services/",
```

Comment:
Ah, I just noticed this was `/registry` and not `/deis/registry`. Disregard - this is reasonable.